### PR TITLE
testmap: update cockpit composer testing

### PR DESCRIPTION
**Pull request by jkozol**
https://github.com/osbuild/cockpit-composer/pull/1568

### CODE REVIEW

File: lib/testmap.py [hunk 1]
```diff
@@ -128,15 +128,16 @@
     },
     'osbuild/cockpit-composer': {
         'main': [
-            'fedora-35',
             'fedora-36',
-            'fedora-36/firefox',
-            'rhel-8-7',
-            'rhel-9-1',
+            'fedora-37',
```
Comment (marusak) [inline lib/testmap.py:132]: Neither of these new tests ran in https://github.com/osbuild/cockpit-composer/pull/1568
Wouldn;t it be better to add them to manual, trigger them in the given PR and then enable them in main?

Comment (jkozol) [reply lib/testmap.py:132]: Do you have any idea why the rhel tests aren't running but fedora-37 is? I just trigger them using my local checkout of bots with the updated testmap. That seems to work fine. Or is that why the tests are stuck pending?

**Comment by marusak** [comment]
I see these context triggered in https://github.com/osbuild/cockpit-composer/pull/1568 and they are red. Marking this as blocked.

**Comment by jkozol** [comment]
@marusak Most tests are passing now but there seems to be some issue with the fedora-37 image. Whatever repo is being used is only pulling osbuld-composer 61 even though 67 is in stable: https://bodhi.fedoraproject.org/updates/FEDORA-2022-d62111bac7

**Comment by marusak** [comment]
We refreshed 37 5 days ago. The update you linked has been pushed to stable 5 days ago as well.

**Comment by jkozol** [comment]
Can you click merge for me?

**Pull request merged**